openedx/core/djangoapps/credit/tasks.py

```diff
@@ -11,12 +11,12 @@
 from celery.utils.log import get_task_logger
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.keys import UsageKey
```

Comment:
combine this import with above import: `from opaque_keys.edx.keys import CourseKey, UsageKey`